include/DREAM/Equations/Fluid/IonBoundaryCondition.tcc

```diff
@@ -0,0 +1,145 @@
+/**
+ * Implementation of a general ion boundary condition, which applies to
+ * all ions and ion charge states in the equation system.
+ */
+
+#include "DREAM/Equations/Fluid/IonBoundaryCondition.hpp"
+
+
+/**
+ * Constructor.
+ */
+template<class T>
+IonBoundaryCondition<T>::IonBoundaryCondition(
+    FVM::Grid *g, IonHandler *ihdl, const len_t iIon,
+    Args&& ... args
```

Comment:
Does this compile? I thought some additional template declarations had to be added for this parameter packing to work?

Reply:
I didn't touch the boundary conditions, I thought this was something that you did because you needed it for STREAM? So, you tell me?

Reply:
Then it probably compiles and there's just some magic which I can't see now going on :)